pyfstat/mcmc_based_searches.py

```diff
@@ -1109,6 +1110,13 @@ def _get_prior_bounds(self, normal_stds=2):
                 prior_bounds[key]["lower"] = (
                     prior_dict["loc"] - normal_stds * prior_dict["scale"]
                 )
+            elif prior_dict["type"] == "lognorm":
+                prior_bounds[key]["lower"] = np.exp(
+                    prior_dict["loc"] - normal_stds * prior_dict["scale"]
+                )
+                prior_bounds[key]["upper"] = np.exp(
+                    prior_dict["loc"] + normal_stds * prior_dict["scale"]
+                )
```

Comment:
To be honest I didn't think about it that carefully... And looking at the last column in my plots, it may not be right. I can check the empirical quantiles.

Reply:
I did a quick check and the argument is sound.

Reply:
confirmed empirically:
```
mu=0.00, sigma=0.00
+-3 sigma quantiles of log(draws):  [0. 0.]
mu+-3*sigma:  [0, 0]
mu=0.00, sigma=0.01
+-3 sigma quantiles of log(draws):  [-0.03006991  0.0297424 ]
mu+-3*sigma:  [-0.03, 0.03]
mu=0.00, sigma=0.10
+-3 sigma quantiles of log(draws):  [-0.29434723  0.30940447]
mu+-3*sigma:  [-0.30000000000000004, 0.30000000000000004]
mu=0.00, sigma=1.00
+-3 sigma quantiles of log(draws):  [-3.08715933  2.84648288]
mu+-3*sigma:  [-3, 3]
mu=1.00, sigma=0.00
+-3 sigma quantiles of log(draws):  [1. 1.]
mu+-3*sigma:  [1, 1]
mu=1.00, sigma=0.01
+-3 sigma quantiles of log(draws):  [0.96850764 1.03160289]
mu+-3*sigma:  [0.97, 1.03]
mu=1.00, sigma=0.10
+-3 sigma quantiles of log(draws):  [0.69705839 1.29209927]
mu+-3*sigma:  [0.7, 1.3]
mu=1.00, sigma=1.00
+-3 sigma quantiles of log(draws):  [-2.00463322  4.12357169]
mu+-3*sigma:  [-2, 4]
mu=10.00, sigma=0.00
+-3 sigma quantiles of log(draws):  [10. 10.]
mu+-3*sigma:  [10, 10]
mu=10.00, sigma=0.01
+-3 sigma quantiles of log(draws):  [ 9.96976993 10.02997379]
mu+-3*sigma:  [9.97, 10.03]
mu=10.00, sigma=0.10
+-3 sigma quantiles of log(draws):  [ 9.70060508 10.30894647]
mu+-3*sigma:  [9.7, 10.3]
mu=10.00, sigma=1.00
+-3 sigma quantiles of log(draws):  [ 6.88668307 12.91323652]
mu+-3*sigma:  [7, 13]
```

Reply:
Ok, fair enough